.github/workflows/bootstrap.yml

```
@@ -15,37 +10,21 @@ on:
       - created
 
 jobs:
-  bootstrap-linux:
-    name: Bootstrap on Linux
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - name: bootstrap.py
-        run: |
-          ghcup config set cache true
-          ghcup install ghc 8.10.7
-          python3 bootstrap/bootstrap.py -w $(ghcup whereis ghc 8.10.7) -d bootstrap/linux-8.10.7.json
-
-      - name: Smoke test
-        run: |
-          _build/bin/cabal --version
-
-      - uses: actions/upload-artifact@v2
-        with:
-          name: cabal-linux-bootstrapped
-          path: _build/artifacts/*
-
-  bootstrap-macos:
-    name: Bootstrap on macOS
-    runs-on: macos-latest
+  bootstrap:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+    name: Bootstrap ${{ matrix.os }}
+    runs-on: $${ matrix.os }}
```

```suggestion
    runs-on: ${matrix.os}
```

that might be the culprit

---

oh dear

---

Thanks @jneira <3

---

Maybe it might be related with self host runners? maybe they can have arbitrary names, idk

---

Yes I think so. and I think somehow that was expanded to something like and empty string, so I didn't notice any bad name anywhere. My fat fingers!